app/src/main/java/com/distillery/interview/util/Event.kt

```diff
@@ -0,0 +1,43 @@
+package com.distillery.interview.util
+
+class Event<out T>(private val content: T) {
```

Comment:
This seems to be useful in MVI arch, but we're using MVVM. Let's just use the `Result`sealed class to handle UI states: Loading, Success, and Error.